rest/taskana-rest-spring-test/src/main/java/pro/taskana/rest/test/SpringSecurityToJaasFilter.java

```diff
@@ -0,0 +1,113 @@
+package pro.taskana.rest.test;
+
+import java.io.IOException;
+import java.security.AccessController;
+import java.util.Optional;
+import javax.security.auth.Subject;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.filter.GenericFilterBean;
+
+import pro.taskana.common.api.exceptions.SystemException;
+import pro.taskana.common.api.security.GroupPrincipal;
+import pro.taskana.common.api.security.UserPrincipal;
+
+/** Simple Filter to map all Spring Security Roles to JAAS-Principals. */
+public class SpringSecurityToJaasFilter extends GenericFilterBean {
```

Comment:
Please mark this class as a duplicate

Reply:
Like this?

/**
 * Simple Filter to map all Spring Security Roles to JAAS-Principals.
 *
 * <p>Duplicate of {@linkplain pro.taskana.rest.test.SpringSecurityToJaasFilter}
 */

Reply:
Like this?

/**
 * Simple Filter to map all Spring Security Roles to JAAS-Principals.
 *
 * <p>Duplicate of {@linkplain pro.taskana.common.rest.SpringSecurityToJaasFilter}
 */